README.md

```diff
@@ -11,7 +11,7 @@ This package integrates Large Language Models (LLMs) into [spaCy](https://spacy.
 - Serializable `llm` **component** to integrate prompts into your pipeline
 - **Modular functions** to define the [**task**](#tasks) (prompting and parsing) and [**backend**](#backends) (model to use)
 - Support for **hosted APIs** and self-hosted **open-source models**
-- Integration with [`MiniChain`](https://github.com/srush/MiniChain) and [`LangChain`](https://github.com/hwchase17/langchain)
```

Comment:
Do I understand it correctly that MiniChain just isn't used much?

Reply:
Yes. We initially used MiniChain as our default backend due to its simplicity. Now that we have our REST-based backend, I don't see an advantage to keeping MiniChain.